src/pages/iou/steps/MoneyRequestConfirmPage.js

```diff
@@ -47,12 +48,28 @@ const propTypes = {
     /** Holds data related to Money Request view state, rather than the underlying Money Request data. */
     iou: iouPropTypes,
 
+    /** The policy of the current request */
+    policy: policyPropTypes,
+
+    policyTags: PropTypes.shape({
+        /** List of tags */
+        tags: PropTypes.arrayOf(PropTypes.string),
```

Comment:
I'm pretty sure this and `categories` should be  objects. They're `{}` in defaultProps, and in Onyx they're objects as well
<img width="1321" alt="image" src="https://github.com/Expensify/App/assets/165133/112ba82e-98fa-40f3-9561-627755d4204c">
<img width="1350" alt="image" src="https://github.com/Expensify/App/assets/165133/159d0085-4a81-46ef-8822-1d6812f36270">